File: loadercomplex-forge/src/ver182/java/com/enderzombi102/loadercomplex/forge18/impl/ForgeServer.java
package com.enderzombi102.loadercomplex.forge18.impl;

import com.enderzombi102.loadercomplex.api.minecraft.server.Server;
import net.minecraft.server.MinecraftServer;

public class ForgeServer implements Server {
	private final MinecraftServer backingServer;

	public ForgeServer(MinecraftServer server) {
		this.backingServer = server;
	}

	@Override
	public Object getObject() {
		return this.backingServer;
	}
}

File: loadercomplex-forge/src/ver122/kotlin/com/enderzombi102/loadercomplex/forge122/impl/server/ForgeServer.kt
package com.enderzombi102.loadercomplex.forge122.impl.server

import com.enderzombi102.loadercomplex.api.minecraft.server.Server
import net.minecraft.server.MinecraftServer

class ForgeServer( private val backingServer: MinecraftServer ) : Server {
	override fun getObject(): Any =
		this.backingServer
}
